README.rst

```diff
@@ -204,39 +204,13 @@ In order to run the unit tests and to get a coverage report, four additional Pyt
   pip3 install --user pytest
   pip3 install --user coverage
 
-
-Reading the docs
-================
```

Comment:
Is it worth mentioning the tutorial folder? Once quickstart is added to it it will start to be a useful documentation source

Reply:
ŵe can update it once we add the quickstart file

Reply:
the user can easaly see the tutorial folder in the repository, that's the first thing that you check

Reply:
The quickstart file should be merged any day now anyway though. What do you think @yguclu ?

Reply:
I still suspect that it might be useful to mention the quickstart file but as it is still not merged I will approve this PR as is

Reply:
OK. In PR #622 we should add a link to the quickstart file in the README.